app/src/main/res/layout/activity_main.xml

```diff
@@ -6,6 +6,52 @@
     android:layout_height="match_parent"
     tools:context=".presentation.ui.MainActivity">
 
+    <com.google.android.material.appbar.MaterialToolbar
+        android:id="@+id/toolbar"
+        android:layout_width="match_parent"
+        android:layout_height="?android:attr/actionBarSize"
+        android:background="@color/colorPrimary"
+        app:title=""
+        app:layout_constraintLeft_toLeftOf="parent"
+        app:layout_constraintRight_toRightOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
```

Comment:
format

Reply:
done. Do i have to extract constraint too?